browser/ui/views/tabs/brave_tab_group_header.cc

```diff
@@ -25,8 +29,25 @@
 #include "ui/views/widget/widget.h"
 #include "ui/views/widget/widget_delegate.h"
 
+BraveTabGroupHeader::BraveTabGroupHeader(TabSlotController& tab_slot_controller,
+                                         const tab_groups::TabGroupId& group,
+                                         const TabGroupStyle& style)
+    : TabGroupHeader(tab_slot_controller, group, style),
+      plus_icon_(AddChildView(views::ImageButton::CreateIconButton(
```

Comment:
Should we make `plus_icon_` as a visible child always?
it's only used for vertical tab's group header.

Reply:
Yup I agree - https://github.com/brave/brave-core/pull/21965#discussion_r1487445093